api/src/main/java/marquez/logging/LoggingJerseyFilter.java

```diff
@@ -0,0 +1,34 @@
+package marquez.logging;
+
+import java.io.IOException;
+import java.util.UUID;
+import javax.ws.rs.container.CompletionCallback;
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import org.slf4j.MDC;
+
+/**
+ * Puts request ID, method, and path to the MDC context so that this information is available from
+ * logs throughout the request context. Request ID is randomly generated UUID, which can be used to
+ * group by to see the logs for a particular request.
+ */
+public class LoggingJerseyFilter implements ContainerRequestFilter, CompletionCallback {
+
+  private static final String REQUEST_ID = "RequestID";
```

Comment:
I would standardize on using _camelCase_ for the fields that are logged. (ex: `requestId`)